rescan.go

```diff
@@ -157,7 +157,7 @@ func (s *ChainService) Rescan(options ...RescanOption) error {
 	// all the options that've been passed in.
 	ro := defaultRescanOptions()
 	ro.endBlock = &waddrmgr.BlockStamp{
-		Hash:   *s.chainParams.GenesisHash,
+		Hash:   chainhash.Hash{},
```

Comment:
What material impact does this have? Seems it has the ability to feed an invalid hash to the end user?

Reply:
Resolved in chat. Merging.